.github/workflows/tests.yml

```diff
@@ -1,290 +1,152 @@
 name: test
 on:
-  push:
-    branches: [ main ]
+  push: { branches: [ main ] }
   pull_request:
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: DoozyX/clang-format-lint-action@v0.13
+      with:
+        source: 'include/ddc/ tests/ examples/'
+        exclude: ''
+        extensions: 'hpp,cpp'
+        clangFormatVersion: 12
+  
   id_repo:
     runs-on: ubuntu-latest
-    outputs:
-      in_base_repo: ${{ steps.identify_repo.outputs.in_base_repo }}
+    outputs: { in_base_repo: '${{ steps.identify_repo.outputs.in_base_repo }}' }
     steps:
     - name: Identify repository
       id: identify_repo
       run: |
         echo "in_base_repo=${{ (github.event == 'push' && github.repository == 'Maison-de-la-Simulation/ddc') || github.event.pull_request.head.repo.full_name == 'Maison-de-la-Simulation/ddc' }}" >> "$GITHUB_OUTPUT"
         echo "$GITHUB_OUTPUT"
 
-  docker-build-test_env:
+  docker-build-env:
+    strategy:
+      matrix: { image: ['test_env', 'jammy_env'] }
```

Comment:
Shouldn't we rename `test_env` by `focal_env` ?

Reply:
yes, but should we keep a focal based image?

Reply:
I guess it provides different compiler versions ? But this may be an indirect advantage. Maybe in the future we should have only one image but with all the desired compilers ?

Reply:
I'd rather remove test_env, switch to jamy as minimum version and add a lunar based image for "latest" compilers

Reply:
I do not see any problem with this.